### ISSUE TITLE
should add double quotation

### ISSUE THREAD

**Issue by omiend** [elastic/elasticsearch#15737]
https://www.elastic.co/guide/en/elasticsearch/reference/current/agg-metadata.html
![image](https://cloud.githubusercontent.com/assets/4135267/12082895/f1d07e7a-b2dc-11e5-934a-e9262bc63bac.png)

elasticsearch/docs/reference/aggregations/misc.asciidoc

```== Aggregation Metadata```

Before
```
[source,js]
--------------------------------------------------
{
    ...
    aggs": {
        "titles": {
            "terms": {
                "field": "title"
            },
            "meta": {
                "color": "blue"
            },
        }
    }
}
--------------------------------------------------
```

After
```
[source,js]
--------------------------------------------------
{
    ...
    "aggs": {
        "titles": {
            "terms": {
                "field": "title"
            },
            "meta": {
                "color": "blue"
            },
        }
    }
}
--------------------------------------------------
```

**Comment by jpountz** [comment]
Fixed via #15739

**Comment by omiend** [comment]
thank you.
